linux-tools/cracklib/cracklib_tests.sh

```diff
@@ -83,7 +84,7 @@ test03()
 {
     tc_register "good password"
 
-    crack_check manjo1234 $DICTDIR/tst_cracklib >$stdout 2>$stderr
+    $PWD/crack_check manjo1234 $DICTDIR/tst_cracklib >$stdout 2>$stderr
     tc_pass_or_fail $? "Unexpected response"
```

Comment:
This will not work if cracklib tests are run from different path, Example: from linux-tools.
Instead of using PWD, better option would be LTPBIN path. In this patch set, this change appears in multiple files. Please correct in all the places. 

Currently in all the scripts, there is reference to LTPBIN, which further needs to be changed as AUTOTESTBIN or an appropriate name.